src/pages/ProfilePage/ProfileComponent/HeaderComponent.js

```diff
@@ -80,7 +124,49 @@ function HeaderComponent() {
             Profile
           </Typography>
           <IconButton edge="end" className={classes.menuButton} color="inherit" aria-label="menu">
-            <MoreVertIcon />
+            {/* <MoreVertIcon /> */}
+            <ShareIcon onClick={handleOpen} />
+            <Modal
+              aria-labelledby="transition-modal-title"
+              className={classes.modal}
+              open={open}
+              onClose={handleClose}
+              closeAfterTransition
+              BackdropComponent={Backdrop}
+              BackdropProps={{
+                timeout: 500,
+              }}
+            >
+              <Fade in={open}>
+                <div className={classes.paper}>
+                  <h2 style={{ textAlign: "center" }}>Share with</h2>
```

Comment:
You can use `typography` here @poojadas07 , this font style doesn't seem to go with the app theme.

Reply:
okay